Gemfile

```diff
@@ -39,6 +39,7 @@ gem 'jbuilder', '~> 2.5'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
```

Comment:
Try not to add comments that don't match the code, but for now lets remove the `byebug` comment and the gem line since we're not using that as a debugger tool.